ng5/src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
```

Comment:
you shouldn't push the environment variables to the repo. Delete them from the repo, add them to the project again and then gitIgnore them :D